mirdata/datasets/compmusic_jingju_acappella.py

```diff
@@ -0,0 +1,362 @@
+"""Jingju A Cappella Singing Dataset Loader
+
+.. admonition:: Dataset Info
+    :class: dropdown
+
+    Description:
+
+    This dataset is a collection of boundary annotations of a cappella singing performed by
+    Beijing Opera (Jingju, 京剧) professional and amateur singers.
+
+    Contents:
+        1. wav.zip: audio files in .wav format, mono or stereo.
+        2. pycode.zip: util code for parsing the .textgrid annotation
+        3. catalogue*.csv: recording metadata, source separation recordings are not included.
+        4. annotation_txt.zip: phrase, syllable and phoneme time boundaries (second) and labels in .txt format
+
+    The annotation_txt.zip folder annotations are represented as follows:
+        1. phrase_char: phrase-level time boundaries, labeled in Mandarin characters
+        2. phrase: phrase-level time boundaries, labeled in Mandarin pinyin
+        3. syllable: syllable-level time boundaries, labeled in Mandarin pinyin
+        4. phoneme: phoneme-level time boundaries, labeled in X-SAMPA
+
+    The boundaries (onset and offset) have been annotated hierarchically:
+        1. phrase (line)
+        2. syllable
+        3. phoneme
+
+    Singing units in pinyin and X-SAMPA have been annotated to a jingju a cappella singing audio dataset.
+
+    The corresponding audio files are the a cappella singing arias recordings, which are stereo or mono,
+    sampled at 44.1 kHz, and stored as .wav files. The .wav files are recorded by two institutes: those file
+    names ending with ‘qm’ are recorded by C4DM, Queen Mary University of London; others file names ending with
+    ‘upf’ or ‘lon’ are recorded by MTG-UPF. Additionally, another collection of 15 clean singing recordings is
+    included in this dataset. They are extracted from the commercial recordings which originally contains karaoke
+    accompaniment and mixed versions.
+
+    Additional details:
```

Comment:
This is good to me already, but if you want to keep the format of previous sections (e.g. contents) I guess you can add an indentation here (feel free to ignor)

Reply:
Cool! I also adapted a bit the whole docstring so it's better organized in sections.